kotlin-mbedtls/src/main/kotlin/org/opencoap/ssl/transport/DtlsServerTransport.kt

```diff
@@ -114,4 +114,6 @@ class DtlsServerTransport private constructor(
         executor.supply {
             dtlsServer.putSessionAuthenticationContext(adr, key, value)
         }
+
+    fun getSessionCid(adr: InetSocketAddress) = dtlsServer.getSessionCid(adr)
```

Comment:
This should also run in the executor, similar to `putSessionAuthenticationContext()`.